src/components/EnvironmentOverride/SecretOverrides.tsx

```diff
@@ -546,7 +555,24 @@ export function OverrideSecretForm({ name, appChartRef, toggleCollapse }) {
                 })
                 setSecretData(json)
             }
-        } catch (error) {}
+            if(isESO && Array.isArray(json?.esoData)){
+                let jsonList = json?.esoData.map((j) => {
+                    let temp = {}
+                    if (j.secretKey) {
```

Comment:
can't we simply return `j` or are there any additional properties that we don't want?

Reply:
New task will change all the working of this function

Reply:
Not making any changes for now